traffic_ops/traffic_ops_golang/profile/profiles.go

```diff
@@ -189,10 +189,20 @@ func (prof *TOProfile) Read(h http.Header, useIMS bool) ([]interface{}, error, e
 	}
 	rows.Close()
 	profileInterfaces := []interface{}{}
+	canReadSecureValue := false
+	if prof.APIInfo() != nil && prof.APIInfo().Version != nil {
```

Comment:
Assign `prof.APIInfo()` to a variable to avoid getting `prof.APIInfo()` so many times.

```go
if apiInfo := prof.APIInfo(); apiInfo != nil && apiInfo.Version != nil {
```